app/javascript/packages/session/requests.ts

```diff
@@ -52,8 +52,6 @@ interface SessionTimedOutStatus {
 
 export type SessionStatus = SessionLiveStatus | SessionTimedOutStatus;
 
-export const SESSIONS_URL = new URL('/api/internal/sessions', window.location.href).toString();
```

Comment:
API endpoints are more consistent than the ones that have locales in them, so it could have been fine if we wanted right? This is just for consistency?

Reply:
The session endpoints don't include any user-facing language currently so there aren't any issues right now, so passing in the URL is just for the localization. The other upside is we get path validation with Rails.

I"m not sure I understand what you mean by API endpoints being more consistent.

Reply:
consistent meaning they don't change based on locale param

Reply:
I had originally knowingly implemented this without the path helper based on the original comment that we don't need the endpoint to be localized, but agree it's nice to use path helpers for validation and having a single source of truth for routes. And avoids having to create an exception.